en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -71,13 +72,13 @@ The figure below shows how you can access your remote receiver (please find the
 ## Spektrum Satellite Receivers
 
 The V5 nano has a dedicated DSM cable.
-The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface.
+The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface. 
 
 
 ## Power
 
 The v5 nano kit includes HV\_PM, which supports 2~10S LiPo battery.
-Please connect the 6pin connector of the HW\_PM module to the flight control `Power1` interface.
+Please connect the 6pin connector of the HW\_PM module to the flight control `Power` interface.
```

Comment:
@cuhome 
- I changed this to `Power` - that is what is on the device right?
- This shows how to power the Flight controller right? Are we assuming that people will not use this with anything that needs to power the servo rail (e.g. fixed wing vehicles)? If not, we should explain how to do this. Does the power module provide a BEC or similar to do this?
- The image below shows the outputs, for which we do NOT describe the power. Any particular reason for this choice?
- Would be great if we could have a note about the use of PM2 here - does not need to be detailed, but we should note the additional power system, and link to more information?

Reply:
The HV_PM module supplies power to the flight controller, but it does not power the servos, and the v5+/v5 nano servo rails cannot provide power to the flight controller. Power2 powered, I may need a separate description.

Reply:
I think you do.  These docs are an OK example of how you might approach this: http://docs.px4.io/master/en/assembly/quick_start_pixhawk4.html#power